sdks/python/apache_beam/examples/snippets/transforms/aggregation/top.py

```diff
@@ -0,0 +1,153 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+
+def top_largest(test=None):
+  # [START top_largest]
+  import apache_beam as beam
+
+  with beam.Pipeline() as pipeline:
+    largest_elements = (
+        pipeline
+        | 'Create numbers' >> beam.Create([3, 4, 1, 2])
+        | 'Largest N values' >> beam.combiners.Top.Largest(2)
+        | beam.Map(print)
+    )
+    # [END top_largest]
+    if test:
+      test(largest_elements)
+
+
+def top_largest_per_key(test=None):
+  # [START top_largest_per_key]
+  import apache_beam as beam
+
+  with beam.Pipeline() as pipeline:
+    largest_elements_per_key = (
+        pipeline
+        | 'Create produce' >> beam.Create([
+            ('🥕', 3),
```

Comment:
although having emoji's in code looks cool, I think it could be problematic for some code editors.

Reply:
Emojis are simply UTF-8 characters, so it's the same as having characters from any non-latin alphabet like Chinese or Cyrillic (which should be tested and supported out of the box for accessibility).

Most modern operating systems and editors handle UTF-8 correctly, even `vi` from a terminal, so I don't think this should be an issue.